cmd/podman/shared/create.go

```diff
@@ -91,7 +91,11 @@ func CreateContainer(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 
 		var healthCheckCommandInput string
 		// if the user disabled the healthcheck with "none", we skip adding it
-		healthCheckCommandInput = c.String("healthcheck-command")
+		healthCheckCommandInput = c.String("health-cmd")
```

Comment:
I'd prefer we error here if both are set, instead of unconditionally overwriting